src/helpers/purchase-operation-helper.ts

```diff
@@ -66,30 +64,6 @@ export class PurchaseFlowError extends Error {
     );
   }
 
-  getPublicErrorMessage(productDetails: Product | null): string {
```

Comment:
Was this function not used?

Reply:
Nope, we are already using the translator for this